rubocop/.rubocop.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  DisplayCopNames: true
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Layout/AlignArguments:
+  EnforcedStyle: with_fixed_indentation
+
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
+Metrics/LineLength:
+  Max: 99
+
+Style/ClassAndModuleChildren:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/TrailingCommaInArguments:
+  EnforcedStyleForMultiline: comma
```

Comment:
This one is `false` everywhere: https://github.com/search?p=1&q=org%3Avinted+TrailingCommaInArguments&type=Code